File: seed/csharp-sdk/exhaustive/explicit-namespaces/src/SeedApi.DynamicSnippets/Example34.cs
using SeedApi;

namespace Usage;

public class Example34
{
    public async Task Do() {
        var client = new SeedApiClient(
            token: "<token>",
            clientOptions: new ClientOptions {
                BaseUrl = "https://api.fern.com"
            }
        );

        await client.EndpointsObject.EndpointsObjectGetAndReturnWithRequiredFieldAsync(
            new TypesObjectWithRequiredField {
                String = "string"
            }
        );
    }

}

File: seed/java-sdk/pagination/default/src/main/java/com/snippets/Example34.java
package com.snippets;

import com.seed.api.SeedApiClient;
import com.seed.api.resources.users.requests.UsersListWithOffsetPaginationRequest;

public class Example34 {
    public static void main(String[] args) {
        SeedApiClient client = SeedApiClient.builder()
                .token("<token>")
                .url("https://api.fern.com")
                .build();

        client.users()
                .listwithoffsetpagination(
                        UsersListWithOffsetPaginationRequest.builder().build());
    }
}
